src/js/tech/html5.js

```diff
@@ -273,13 +273,26 @@ class Html5 extends Tech {
       return;
     }
     const listeners = {
-      change(e) {
-        techTracks.trigger({
+      change: (e) => {
```

Comment:
this was changed to an arrow function so that line 294 can be used as is without requiring to get the remoteText getter in the higher scope as we only need it in one usecase.